reforger/index.html

```diff
@@ -0,0 +1,100 @@
+---
+title: Arma Reforger | Swedish Forces Pack
+images:
+  -
+    path: images/ak4or.png
+    title: Marksman
+  -
+    path: images/ksp58.png
+    title: Machinegunner
+  -
+    path: images/leaders.png
+    title: Squadleaders
+  -
+    path: images/pskott.png
+    title: Antitank soldier
+
+---
+
+<div class="container">
+  <h1>Swedish Forces Pack</h1>
+</div>
+
+<div class="container">
+  <h2>Description</h2>
+
+  <p>
+   The original Swedish Armed Forces experience for ArmA are now entering the next generation with the first release for Arma Reforger.
+  </p>
+
+  <p>
+    SFP is now leaving the previous games and relases behind and continue with 100% new content made for Reforger with professional tools with the highest quality. <br/>
+    Focus is infantry and the era (1988-1992) represented in Reforger. 
+  </p>
+
+  <p>
+   Our first release comes with a Conflict version and a coop in order to test our mod, but more missions are planned, both cooperative and competitive with the hopes to bring players from ArmA3 to Reforger as a preperation for ArmA4. 
```

Comment:
Fix indentation. Arma, not ArmA
```suggestion
    Our first release comes with a Conflict version and a coop in order to test our mod, but more missions are planned, both cooperative and competitive with the hopes to bring players from Arma 3 to Reforger as a preperation for Arma 4. 
```